tests/ui/test_login.py

```diff
@@ -1,23 +1,24 @@
+import os
 import pytest
 
 from pages.desktop.home import Home
 
 
-@pytest.mark.skip(
-    reason='https://github.com/mozilla/addons-server/issues/2462')
-def test_login(base_url, selenium, user):
+@pytest.mark.skipif(os.environ.get('PYTEST_BASE_URL') is None,
+                    reason='Live Server login currently not functioning')
```

Comment:
Also, this should probably use another environment variable than `PYTEST_BASE_URL` to skip and switch between live_server and real-environments (e.g ``UITEST_USE_LIVESERVER`` or so) to be more explicit about any implications it has on the tests.

Also, can be changed later but we should abstract these switches and skips into something like

```python
requires_real_environment = pytest.mark.skipif(os.environ.get('UITEST_USE_LIVESERVER') is None, reason='Requires a real environment not the live_server')

@requires_real_environment
def test_login(my_base_url, selenium, user):
    # ...
```

And set the actual reason in the docstring. Again, let's not do this now but maybe later when we have more tests that need switching back and forth.